dockerfiles/build_manylinux_x86_64.Dockerfile

```diff
@@ -94,3 +94,19 @@ RUN which gcc && gcc --version && \
 # We use the wildcard option to disable the checks. This was added
 # in git 2.35.3
 RUN git config --global --add safe.directory '*'
+
+######## vcpkg ########
+WORKDIR /opt
+ENV VCPKG_HASH="3f5ad7be7693ce6ac5599ddb7cc24f260b9d44f9"
+COPY install_vcpkg.sh ./
+RUN yum install -y zip unzip
+RUN ./install_vcpkg.sh "${VCPKG_HASH}"
```

Comment:
Similarly, add a comment here saying that vcpkg is used to build some dependencies that are not yet vendored into the build system (so we know when this is safe to remove)

Reply:
Added a comment here as well.